src/Components/Firebase/Collection.js

```diff
@@ -31,7 +31,8 @@ class Collection {
     try {
       const docRef = await this.collectionRef.add({
         ...data,
-        created: this.getServerTimestamp()
+        created: this.getServerTimestamp(),
```

Comment:
When I first create a new project, this appears as 'NA' - and if I duplicate a newly created project, both fields flicker with real values, and then revert back to NA. I'm not sure why this'd be a bug (everything works if I navigate away from the page, and then navigate back). 

<img width="732" alt="Screenshot 2021-04-09 at 08 24 24" src="https://user-images.githubusercontent.com/22618458/114145298-28eacb00-990e-11eb-83dd-bf7ad540c134.png">

Reply:
Thanks! Looks like I wasn't actually returning all the fields when getting from firestore :)